ibis/expr/window.py

```diff
@@ -245,6 +248,7 @@ def _replace(self, **kwds):
             order_by=kwds.get('order_by', self._order_by),
             preceding=kwds.get('preceding', self.preceding),
             following=kwds.get('following', self.following),
+            max_lookback=kwds.get('max_lookback', self.max_lookback),
```

Comment:
Can you add a test for `_replace`? You can make a `group_by` or `order_by` call on a window to exercise this code path.

Reply:
Yes, added tests in next commit and also fixed equals() to account for max_lookback.